widgetastic_patternfly.py

```diff
@@ -93,6 +93,28 @@ def title(self):
         return self.browser.get_attribute('title', self)
 
 
+class ViewChangeButton(Widget, ClickableMixin):
+    """A PatternFly/Bootstrap view selection button in CFME 56z
+
+        .. code-block:: python
+
+            Button(title='Show xyz')
+            assert button.active
+    """
+    CHECK_VISIBILITY = True
+
+    def __init__(self, parent, title, **kwargs):
+        Widget.__init__(self, parent, logger=kwargs.pop('logger', None))
+        self.title = title
+
+    def __locator__(self):
+        return './/a[(@title={})]/i[contains(@class, "fa")]/..'.format(quote(self.title))
+
+    @property
+    def active(self):
+        return 'active' in self.browser.classes(self.parent)
```

Comment:
Yep, the anchor element doesn't have the active class, the anchors are in a list and the <li> parent element holds the 'active' class.